lib/s3/list-objects.js

```diff
@@ -9,10 +9,10 @@ export default async function (bucket, folder) {
     Bucket: `hyper-queue-${bucket}`,
     Prefix: folder,
   }).then(compose(
-    map(replace('.json', '')),
+    map(replace(".json", "")),
     reject(equals(`${folder}/`)),
```

Comment:
Does the folder itself in s3 come back as a key? May need to do this in the s3 adapter then?

Reply:
It could possibly!